### PR TITLE
Update project-recommendations.md

### PR DESCRIPTION
**Pull request by mirnawong1**
add info about package being base version and doesn't import customization. raised here; https://dbt-labs.slack.com/archives/C04HJR9B69H/p1720019114070189?thread_ts=1719937667.452869&cid=C04HJR9B69H

### CODE REVIEW

File: website/docs/docs/collaborate/project-recommendations.md [hunk 1]
```diff
@@ -4,12 +4,13 @@ sidebar_label: "Project recommendations"
...
  
-dbt Explorer provides recommendations about your project from the `dbt_project_evaluator` [package](https://hub.getdbt.com/dbt-labs/dbt_project_evaluator/latest/) using metadata from the Discovery API.
+dbt Explorer provides recommendations about your project from the `dbt_project_evaluator` [package](https://hub.getdbt.com/dbt-labs/dbt_project_evaluator/latest/) using metadata from the Discovery API. 
 
-Explorer also offers a global view, showing all the recommendations across the project for easy sorting and summarizing. 
-
-These recommendations provide insight into how you can build a more well documented, well tested, and well built project, leading to less confusion and more trust.
+- Explorer also offers a global view, showing all the recommendations across the project for easy sorting and summarizing.
+- These recommendations provide insight into how you can create a better-documented, better-tested, and better-built dbt project, leading to less confusion and more trust.
+- The recommendations are pre-defined and don't import any customizations applied to your package or project.
```
Comment (mirnawong1) [inline website/docs/docs/collaborate/project-recommendations.md]:
```suggestion
- For seamless and consistent experience, recommendations are pre-defined, and don't import any customizations applied to your package or project.
```

File: website/docs/docs/collaborate/project-recommendations.md [hunk 2]
```diff
@@ -4,12 +4,13 @@ sidebar_label: "Project recommendations"
...
  
-dbt Explorer provides recommendations about your project from the `dbt_project_evaluator` [package](https://hub.getdbt.com/dbt-labs/dbt_project_evaluator/latest/) using metadata from the Discovery API.
+dbt Explorer provides recommendations about your project from the `dbt_project_evaluator` [package](https://hub.getdbt.com/dbt-labs/dbt_project_evaluator/latest/) using metadata from the Discovery API. 
 
-Explorer also offers a global view, showing all the recommendations across the project for easy sorting and summarizing. 
-
-These recommendations provide insight into how you can build a more well documented, well tested, and well built project, leading to less confusion and more trust.
+- Explorer also offers a global view, showing all the recommendations across the project for easy sorting and summarizing.
+- These recommendations provide insight into how you can create a better-documented, better-tested, and better-built dbt project, leading to less confusion and more trust.
+- For seamless and consistent experience, recommendations are pre-defined, and don't import any customizations applied to your package or project.
```
Comment (mirnawong1) [inline website/docs/docs/collaborate/project-recommendations.md]:
```suggestion
- For seamless and consistent experience, recommendations are pre-defined and don't import customizations applied to your package or project.
```

File: website/docs/docs/collaborate/project-recommendations.md [hunk 3]
```diff
@@ -37,7 +39,7 @@ The Recommendations overview page includes two top-level metrics measuring the t
 
 ## The Recommendations tab
 
-Models, sources and exposures each also have a Recommendations tab on their resource details page, with the specific recommendations that correspond to that resource:
+Models, sources, and exposures each also have a Recommendations tab on their resource details page, with the specific recommendations that correspond to that resource:
```
Comment (mirnawong1) [inline website/docs/docs/collaborate/project-recommendations.md]:
```suggestion
Models, sources, and exposures each also have a **Recommendations** tab on their resource details page, with the specific recommendations that correspond to that resource:
```

**Review by nataliefiann — COMMENTED** [review]
Hiya @mirnawong1 

This is great! I have no changes or suggestions on my end. I haven't approved it for you (as Cameron may have some thoughts) but happy to do so if you'd like me to. Please let me know if there's anything else I can do to help with this.

Kind Regards

File: website/docs/docs/collaborate/project-recommendations.md [hunk 4]
```diff
@@ -4,12 +4,13 @@ sidebar_label: "Project recommendations"
...
  
-dbt Explorer provides recommendations about your project from the `dbt_project_evaluator` [package](https://hub.getdbt.com/dbt-labs/dbt_project_evaluator/latest/) using metadata from the Discovery API.
+dbt Explorer provides recommendations about your project from the `dbt_project_evaluator` [package](https://hub.getdbt.com/dbt-labs/dbt_project_evaluator/latest/) using metadata from the Discovery API. 
 
-Explorer also offers a global view, showing all the recommendations across the project for easy sorting and summarizing. 
-
-These recommendations provide insight into how you can build a more well documented, well tested, and well built project, leading to less confusion and more trust.
+- Explorer also offers a global view, showing all the recommendations across the project for easy sorting and summarizing.
+- These recommendations provide insight into how you can create a better-documented, better-tested, and better-built dbt project, leading to less confusion and more trust.
+- For seamless and consistent experience, recommendations are pre-defined and don't import customizations applied to your package or project.
```
Comment (mirnawong1) [inline website/docs/docs/collaborate/project-recommendations.md]:
```suggestion
- For a seamless and consistent experience, recommendations uses ` dbt_project_evaluator`'s pre-defined settings and don't import customizations applied to your package or project.
```

File: website/docs/docs/collaborate/project-recommendations.md [hunk 5]
```diff
@@ -4,12 +4,13 @@ sidebar_label: "Project recommendations"
 description: "dbt Explorer provides recommendations that you can take to improve the quality of your dbt project."
 ---
  
-dbt Explorer provides recommendations about your project from the `dbt_project_evaluator` [package](https://hub.getdbt.com/dbt-labs/dbt_project_evaluator/latest/) using metadata from the Discovery API.
+dbt Explorer provides recommendations about your project from the `dbt_project_evaluator` [package](https://hub.getdbt.com/dbt-labs/dbt_project_evaluator/latest/) using metadata from the Discovery API.
```
Comment (nataliefiann) [inline website/docs/docs/collaborate/project-recommendations.md]: Hiya @mirnawong1 

This reads really well IMO. A possible suggestion and non-blocker - could Discovery API be hyperlinked to the Discovery API page? (https://docs.getdbt.com/docs/dbt-cloud-apis/discovery-api)

Kind Regards
Natalie

Comment (mirnawong1) [reply website/docs/docs/collaborate/project-recommendations.md]: good shout! yes will add!

Comment (mirnawong1) [inline website/docs/docs/collaborate/project-recommendations.md]:
```suggestion
dbt Explorer provides recommendations about your project from the `dbt_project_evaluator` [package](https://hub.getdbt.com/dbt-labs/dbt_project_evaluator/latest/) using metadata from the [Discovery API](/docs/dbt-cloud-apis/discovery-api). 
```

File: website/docs/docs/collaborate/project-recommendations.md (hunk 4 above)
Comment (nataliefiann) [reply website/docs/docs/collaborate/project-recommendations.md]: Hiya @mirnawong1, I think the previous sentence was really effective and well explained. It flowed really nicely.
```suggestion
- For seamless and consistent experience, recommendations are pre-defined and don't import customizations applied to your package or project..
```

File: website/docs/docs/collaborate/project-recommendations.md (hunk 5 above)
Comment (nataliefiann) [reply website/docs/docs/collaborate/project-recommendations.md]: Hiya @mirnawong1 

For some reason, I couldn't add a suggestion to this.
For the sentence, **For seamless and consistent experience, recommendations are pre-defined and don't import customizations applied to your package or project** flowed nicely and read really well.

Kind Regards

**Review by nataliefiann — COMMENTED** [review]
Hiya @mirnawong1 

I still think this is an excellent piece. I did add a suggestion and non-blocker but I don't think anything really needs changing. I hope this helps. Let me know if there's anything else I can do to help with this.

Kind Regards
Natalie

File: website/docs/docs/collaborate/project-recommendations.md (hunk 4 above)
Comment (mirnawong1) [reply website/docs/docs/collaborate/project-recommendations.md]: Thanks @nataliefiann ! Do  you m we a the sentence that doesn't mention project evaluator? I see what you mean and i wanted to clarify or be more explicit in the exact predefined a settings that recommendation uses. I thought previously it wasn't as clear

**Comment by mirnawong1** [comment]
hey @cafzal , just pinging you to see if you're ok with the changes here. thank you!

**Comment by mirnawong1** [comment]
merging this!

**Comment by mirnawong1** [comment]
preview: https://docs-getdbt-anxonmujc-dbt-labs.vercel.app/docs/collaborate/project-recommendations

**Pull request merged**